tests/UI/pages/BO/orders/add.js

```diff
@@ -864,6 +864,7 @@ class AddOrder extends BOBasePage {
    */
   async setRecycledPackaging(page, isEnabled) {
     await this.setChecked(page, this.recycledPackagingToggleInput(isEnabled ? 1 : 0));
+    await page.waitForTimeout(2000);
```

Comment:
There is no js events to wait for ?

Reply:
it's the same event for all delivery settings (all switch buttons)
![Capture d’écran de 2022-08-25 15-59-05](https://user-images.githubusercontent.com/24647495/186684965-98740b00-0459-49cc-b7fe-f9baf6b4050e.png)